src/txn/txn_rollback_to_stable.c

```diff
@@ -1213,6 +1205,14 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
             WT_ERR_NOTFOUND_OK(ret, false);
         }
         max_durable_ts = WT_MAX(newest_start_durable_ts, newest_stop_durable_ts);
+
+        if (F_ISSET(S2C(session), WT_CONN_RECOVERING) &&
+          txn_global->stable_timestamp == WT_TS_NONE) {
```

Comment:
This condition is wrong. we are ignoring RTS during recovery when stable timestamp is not set. we should add an additional check to the above condition the table has some updates in it with a timestamp.

Reply:
Fixed the condition and added an extra condition to skip tables without checkpoint information.